java/src/jmri/jmrit/operations/rollingstock/cars/CarsTableModel.java

```diff
@@ -259,6 +259,7 @@ synchronized void updateList() {
         return getCarList(_sort);
     }
 
+    @edu.umd.cs.findbugs.annotations.SuppressFBWarnings(value = "DB_DUPLICATE_SWITCH_CLAUSES")
```

Comment:
@DanielBoudreau I quote from that page:
>Note that this should be all on one line. Although Java itself considers it optional, we require the second "justification" argument. Explaining why you've added this annotation to suppress a message will help whoever comes after you and is trying to understand the code. It will also help make sure you properly understand the cause of the underlying bug report: Sometimes what seems a false positive really isn't. Annotations without a justification clause will periodically be removed.

So, this suppression (and all others in this PR) are liable for removal at a later date if no justification is included, even if it _seems_ obvious.